src/app/components/Transactions/LogEvent.tsx

```diff
@@ -111,13 +112,14 @@ const DecodedLogEvent: FC<{ scope: SearchScope; event: RuntimeEvent }> = ({ scop
     case RuntimeEventType.accountstransfer:
     case RuntimeEventType.consensus_accountsdeposit:
     case RuntimeEventType.consensus_accountswithdraw:
-    default:
       return (
         <div>
           <div>{eventName}</div>
           <pre>{JSON.stringify(event, null, ' ')}</pre>
         </div>
       )
+    default:
+      throw new ExhaustedTypeError('Unexpected event type', event.type)
```

Comment:
I'm not happy with throwing an error (and hence breaking the whole  page) just because we see something we don't recognize. I thing we should also have an `exhaustedTypeWarning()` function, which just uses `console.warn()`, and use that in the milder cases like this one.

Reply:
Converted to warnings in production and errors in development